cli/pcluster/config/param_types.py

```diff
@@ -222,8 +207,71 @@ def __ne__(self, other):
     def _value_eq(self, other):
         return self.value == other.value
 
+    def get_storage_key(self):
```

Comment:
Yes. We could think about unifying these attributes; this would probably mean renaming the ``cfn_mapping_parameter``, which I see is very much used. In that case, I'd prefer doing it as a separate PR if you agree.